syntax = "proto3";

package mprpc;

message RpcHeader
{
    string server_name = 1;
    string method_name = 2;
    uint32 args_size = 3;
}